app/assets/stylesheets/events.scss

```diff
@@ -14,3 +22,202 @@
 #new-position-form {
   display: none;
 }
+
+/*             */
+/* Top section */
+/*             */
+
+.title {
+  height: auto;
+
+  h1 {
+    font-family: $brand-font;
+    color: $main-color-5;
+  }
+}
+
+#new-event-button {
+  background: none;
+  color: $main-color-5;
+  border: 1px solid;
+  border-color: $main-color-5;
+}
+
+/*      */
+/* List */
+/*      */
+
+#event-list-panel {
+  height: 40em;
+  overflow: auto;
+  background: none;
+  padding: 0;
+}
+
+.events-table {
+  background: none;
+  margin-top: 0.5rem;
+
+  th {
+    color: $main-color-5;
+    text-align: center;
+  }
+
+  tr td div.event-info {
+    position: relative;
+    min-height: 2.5rem;
+
+    div.event-times {
+      font-size: 7.5pt;
+      font-style: italic;
+      text-align: right;
+      position: absolute;
+      bottom: 0;
+      right: 0;
+    }
+  }
+
+  tr td.venue-info {
+    text-align: center;
+    font-size: 8pt;
+  }
+
+  tr td, a {
+    color: $main-color-5;
+    line-height: 0.725rem;
+  }
+
+  a {
+    font-size: 10pt;
+
+  }
+
+  a:hover {
+    color: orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.